src/lib/Notification.js

```diff
@@ -23,7 +23,7 @@ class Notification {
 
         this.notificationEl.innerHTML = `
             <span id="${uniqueLabel}"></span>
-            <button class="close-btn" type="button">${__('notification_button_default_text')}</button>
+            <button class="close-btn" type="button">✕</button>
```

Comment:
Should we use an icon here rather than a unicode character? Should there be an aria-label or something for accessibility?

Reply:
Will introduce an SVG icon thanks